.ci/lint_r_code.R

```diff
@@ -86,7 +86,7 @@ LINTERS_TO_USE <- list(
     , "unneeded_concatenation" = lintr::unneeded_concatenation_linter
 )
 
-print(sprintf("Found %i R files to lint\n", length(FILES_TO_LINT)))
+noquote(paste0("LINT INFO: ", length(FILES_TO_LINT), " R files need linting"))
```

Comment:
```suggestion
noquote(paste0(length(FILES_TO_LINT), " R files need linting"))
```

I don't think the `LINT INFO` is adding any information, so I'd like to suggest removing it

Reply:
ok got it.  I'll make the change